bin/runtime-common/src/messages_benchmarking.rs

```diff
@@ -43,6 +43,29 @@ use sp_std::{fmt::Debug, prelude::*};
 use sp_trie::{record_all_keys, trie_types::TrieDBMutV1, LayoutV1, MemoryDB, Recorder, TrieMut};
 use sp_version::RuntimeVersion;
 
+/// Return this chain account, used to dispatch message.
```

Comment:
This change isn't actually required, it's just a small refactoring I've made when working on this issue - to make it clear that this account is used to dispatch incoming messages